### PR TITLE
Issue 1666 - future proofing smart contracts

### PR DESCRIPTION
**Pull request by ElPaisano**
Addresses [ issue 1666](https://github.com/filecoin-project/filecoin-docs/issues/1666).

Created a new page under Developers -> Smart Contracts -> How Tos based off of https://github.com/filecoin-project/testnet-hyperspace/issues/9

https://bafybeigkiflpaabs5jwrcwk4uonkp57eezj4wuovouxe7ih47ehjibx46y.on.fleek.co/developers/smart-contracts/how-tos/future-proof/

### CODE REVIEW

**Comment by ElPaisano** [comment]
@johnnymatthews can you give this a review and let me know if I'm on the right track here?

File: content/en/developers/smart-contracts/how-tos/future-proof/index.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    parent: "developers-how-tos"
+toc: true
+images: []
+aliases:
+    - "/fvm/how-tos/future-proof-a-contract/"
+---
+
+{{< beta-warning >}}
+
+**Developers should take the time to thoroughly read through the following summary of possible contract future-proofing updates, as failure to properly future proof smart contracts may result in incompatibility with the [v1.20.0-hyperspace-nv20 release](https://github.com/filecoin-project/testnet-hyperspace/issues/9) and future Filecoin releases.**
```
Comment (johnnymatthews) [inline content/en/developers/smart-contracts/how-tos/future-proof/index.md:19]: Maybe add this into an alert shortcode? So something like:

```suggestion
{{< alert >}}
Developers should take the time to thoroughly read through the following summary of possible contract future-proofing updates, as failure to properly future proof smart contracts may result in incompatibility with the [v1.20.0-hyperspace-nv20 release](https://github.com/filecoin-project/testnet-hyperspace/issues/9) and future Filecoin releases.
{{< /alert >}}
```

**Review by johnnymatthews — CHANGES_REQUESTED** [review]
Looks good. Just need to address a couple of comments I left.

File: content/en/developers/smart-contracts/how-tos/future-proof/index.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+---
+title: "Future-proof a contract"
+description: "Learn how to future-proof an existing smart contract for compatibility with the v1.20.0-hyperspace-nv20 release."
+lead: "Learn how to future-proof an existing smart contract for compatibility with the `v1.20.0-hyperspace-nv20` release and future network upgrades."
```
[the remark this answers is not in the document]
Comment (ElPaisano) [reply content/en/developers/smart-contracts/how-tos/future-proof/index.md]: Removed in https://github.com/filecoin-project/filecoin-docs/pull/1691/commits/9d3935f4918953bfa602a8a17345722b17cfdf95

**Comment by ElPaisano** [comment]
> Looks good. Just need to address a couple of comments I left.

Alright comments addressed, should be good to merge

**Review by johnnymatthews — APPROVED** [review]

**Comment by ElPaisano** [comment]
@johnnymatthews I don't have merge capabilities even with the approval

image

**Pull request merged**